healthcheck/healthcheck.go

```diff
@@ -98,7 +99,7 @@ func (healthCheck *HealthCheck) isHealthy() error {
 	if dbErr := db.WithinTx(healthCheck.DataStore, func(transaction transaction.Transaction) error { return nil }); dbErr != nil {
 		return dbErr
 	}
-	if _, syncErr := healthCheck.Etcd.Fetch(healthCheck.etcdKey); syncErr != nil && syncErr != etcdv3.KeyNotFound {
+	if _, syncErr := healthCheck.Etcd.Fetch(context.Background(), healthCheck.etcdKey); syncErr != nil && syncErr != etcdv3.KeyNotFound {
```

Comment:
It would be good to set here reasonable timeout. If etcd is slow it means that Gohan is not responsive too. Especially with this PR.

Reply:
this PR did not changed anything w.r.t. healthcheck. unless explicitly configured, etcd uses a [default timeout of 1s](https://github.com/cloudwan/gohan/blob/master/sync/util/etcd.go#L25) for all operations (including Fetch).

if etcd gets unresponsive, it's an issue per se, and should be fixed instead of being covered up.

at the same, I agree that it makes sense to pass a context here.